node/src/actors/json_rpc/json_rpc_methods.rs

```diff
@@ -1098,17 +1098,55 @@ pub async fn data_request_report(params: Result<(Hash,), jsonrpc_core::Error>) -
         .await
 }
 
+/// Params of getBlockChain method
+#[derive(Debug, Default, Deserialize, Serialize)]
+pub struct GetBalanceParams {
+    /// Public key hash
+    #[serde(default)] // default to default pkh
```

Comment:
This field should not use `#[serde(default)]` because it is a required argument.

```suggestion
```